participant/styles/_uswds-theme-custom-styles.scss

```
@@ -41,11 +41,28 @@ i.e.
   flex: 1;
 }
 
+
 // Custom classes
 p {
   @include typeset('body');
 }
 
+// File Input styling
```

Nit: Would be more consistent with the rest of the file to put this section after the custom css classes with a comment like this:

```suggestion
// USWDS File Input:
// - Fix issue where CSS captures mouse clicks
// - Change styling
```